.github/workflows/pr-test.yml

```diff
@@ -7,26 +7,14 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
consider consolidating test and build into a single workflow. Example:
https://github.com/hypertrace/service-framework/pull/70/files

Reply:
Sure. Will merge them in separate PR.